sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py

```diff
@@ -267,19 +212,22 @@ def fix_big_data_passing(
                     inputs_directly_consumed_as_parameters.add(
                         (template_name, input_name))
                 elif parts[1] == 'artifacts':
-                    raise AssertionError(
+                    # raise RuntimeError(
+                    logging.error(
```

Comment:
We may want to convert these 3 log errors back to `RuntimeError` if this is a potential problem in our code, or, use `assert(...)` if this is a problem with user provided input. I had temporarily "down-graded" those to log messages in order to get the compilation to complete and only print the errors for debugging.

Reply:
Addressed